checkov/common/sast/enums.py

```diff
@@ -0,0 +1,14 @@
+
+from enum import Enum
+
+
+class ExtendedEnum(Enum):
+    @classmethod
+    def list(cls):
+        return list(map(lambda c: c.value, cls))
```

Comment:
Why do we need this extended enum? Wouldn't it be enough to implement `list(cls)` in `SastLanguages`?